dev-packages/e2e-tests/package.json

```diff
@@ -23,7 +23,8 @@
     "esbuild": "0.20.0",
     "glob": "8.0.3",
     "ts-node": "10.9.1",
-    "yaml": "2.2.2"
+    "yaml": "2.2.2",
+    "rimraf": "^5.0.0"
```

Comment:
not sure why but without adding rimraf explicitly here, I can't use the `yarn test:run` command to run e2e tests locally 🤔